crates/astria-conductor/src/executor/mod.rs

```diff
@@ -128,12 +132,15 @@ impl Executor {
             .await
             .wrap_err("executor failed to get commitment state")?;
 
+        let executable_block_height = commitment_state.soft.number + init_sequencer_height;
```

Comment:
this is only correct if we are executing empty blocks - should we update conductor to always execute empty blocks? otherwise this will not work

Reply:
pr to remove disable empty block execution here: https://github.com/astriaorg/astria/pull/556